docs/plugins/lamp-plugin.md

```diff
@@ -27,6 +27,23 @@ The Probo [Drupal plugin](/plugins/drupal-plugin/), [WordPress plugin](/plugins/
 | `phpConstants` | A hash of constants, such as {const1: 'const1Value', const2: 'const2Value',}. This will overwrite any other auto_prepend_file directives in your php.ini. Accepts a **hash** value. |
 | `phpMods` | An array of php5 modules to enable (should be installed via installPackages if needed). Accepts an **array** value. |
 
+### Modify PHP Environment Options in a Build
```

Comment:
I feel like we can remove this header and then move the content above the table of options. That way we are introducing the concepts before jumping into the technical details. Also that's one less header for people to digest while skimming the page. What do you think?